code/game/machinery/vending.dm

```diff
@@ -1003,21 +1005,23 @@
 
 /obj/machinery/vending/security
 	name = "\improper SecTech"
-	desc = "A security equipment vendor."
-	product_ads = "Crack capitalist skulls!;Beat some heads in!;Don't forget - harm is good!;Your weapons are right here.;Handcuffs!;Freeze, scumbag!;Don't tase me bro!;Tase them, bro.;Why not have a donut?"
+	desc = "A security equipment vendor, with some customary donuts; help yourself."
+	product_ads = "Defend yourself!;Help others!;Don't forget - security is a need not a want!;Your weapons are right here.;Handcuffs!;Freeze, scumbag!;Don't tase me bro!;Tase them, bro.;Why not have a donut?"
```

Comment:
"Don't forget - security is a need not a want!" is fucking stupid. Change the other meme-y quotes to be more befitting as well.